src/Common/src/System/Net/SocketAddressPal.Windows.cs

```diff
@@ -18,6 +18,14 @@ public static unsafe AddressFamily GetAddressFamily(byte[] buffer)
 
         public static unsafe void SetAddressFamily(byte[] buffer, AddressFamily family)
         {
+            if ((int)(family) > (int)AddressFamily.Max)
+            {
+                // For legacy values up to AddressFamily.Max, family maps directly to Winsock value.
+                // Other values will need mapping if/when supported.
+                // Currently, that is Netlink, Packat and ControllerAreaNetwork, neither of them supported on Windows.
+                throw new PlatformNotSupportedException();
+            }
```

Comment:
I assume there are no similar changes that need to be done anywhere for ProtocolFamily because we don't actually use ProtocolFamily anywhere?

Reply:
I did search through the code base and I did not find anything obvious for ProtocolFamily.